store/src/test/java/org/apache/rocketmq/store/queue/BatchConsumeMessageTest.java

```diff
@@ -290,7 +295,8 @@ public void testDispatchBuildBatchConsumeQueue() throws Exception {
                 timeStart = putMessageResult.getAppendMessageResult().getStoreTimestamp();
             }
             if (i == 30) {
-                timeMid = putMessageResult.getAppendMessageResult().getStoreTimestamp();;
+                timeMid = putMessageResult.getAppendMessageResult().getStoreTimestamp();
+                ;
```

Comment:
It would be better remove this useless line

Reply:
> It would be better to remove this useless line

Done